client/packages/common/src/ui/layout/tables/material-react-table/useTableDisplayOptions.tsx

```diff
@@ -148,13 +165,22 @@ export const useTableDisplayOptions = <T extends MRT_RowData>(
           : 'inherit',
         fontStyle: row.getCanExpand() ? 'italic' : 'normal',
         cursor: onRowClick ? 'pointer' : 'default',
+        '&.Mui-selected': {
+          backgroundColor: 'background.secondary',
+        },
+        '& td::after': {
+          backgroundColor: 'transparent',
+        },
+        // TO-DO: Figure out how to target the "hover" state for these rows
```

Comment:
I spent way too long trying to modify the background color when rows are both selected and hovered, with no joy. Open to suggestions, otherwise might have to leave this for now.

Reply:
https://github.com/msupply-foundation/open-msupply/compare/9181-style-tweaks...9181-style-tweaks(selected-row-colour) ?